spec/game/game_spec.rb

```diff
@@ -16,106 +18,109 @@ module Game
     end
 
     it "allows tokens to be placed on a board" do
-      game = @game.make_move(1, 1)
+      game = @game.place_token(1, 1)
       expect(game.object_id).not_to eq @game.object_id
       expect(game.state).to eq :ok
       expect(game.current_board).to eq ['X', *2..9]
       expect(game.available_positions).to eq [*2..9]
     end
 
     it "does not allow tokens to be placed on top of other tokens" do
-      game = @game.make_move(1, 1)
-      game = game.make_move(2, 1)
+      game = @game.place_token(1, 1)
+      game = game.place_token(2, 1)
       expect(game.state).to eq :bad_position
       expect(game.current_board).to eq ['X', *2..9]
       expect(game.available_positions).to eq [*2..9]
     end
 
     [-1, 10, "a", "A", "pos", "£", " ", "", nil].each do |position|
       it "does not allow tokens to be placed in invalid position of '#{position}''" do
-        game = @game.make_move(1, position)
+        game = @game.place_token(1, position)
         expect(game.state).to eq :bad_position
       end
     end
 
     it "after one move player two is the current player" do
-      game = @game.make_move(1, 1)
+      game = @game.place_token(1, 1)
       expect(game.current_player).to be 2
     end
 
     it "after player one and two make a move player one is the current player" do
-      game = @game.make_move(1, 1)
-      game = game.make_move(2, 2)
+      game = @game.place_token(1, 1)
```

Comment:
Could your game keep track of which player's turn it is instead of pushing that responsibility to whoever is using the game? That would simplify how the game is used.

Reply:
Yes, that makes a lot of sense.